### DISCUSSION TITLE
Help with github pages

### DISCUSSION THREAD

**Discussion by CoderGuyTedd** [community/community#139202] [New to GitHub]
How do i make a github page for my profile, like teddcoderguy.github.io instead of teddcoderguy.github.io/teddwebsite

**Comment by sam-math** [comment]
Check this: [https://pages.github.com/](https://pages.github.com/)

**Comment by davevad93** [comment]
Hi @CoderGuyTedd ,

To do this, you must create a repository named `<username>.github.io` , which in your case would be `CoderGuyTedd.github.io`.

For more info, take a look at the docs:

- [Types of GitHub Pages sites](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites)

**Reply by CoderGuyTedd** [reply]
thanks

**Reply by davevad93** [reply]
You're welcome @CoderGuyTedd  👍
